test/functionalTest/harnessFunctions.sh

```diff
@@ -378,26 +374,24 @@ function brokerStart()
 #
 function brokerStop
 {
-  if [ "$1" == "-v" ]
-  then
-    _verbose=1
-    shift
-  fi
-
   role=$1
-  if [ "$role" == "" ]
-  then
-    role=CB
-  fi
- 
-  vMsg "Stopping broker $1"
 
   if [ "$role" == "CB" ]
   then
     pidFile=$CB_PID_FILE
     port=$CB_PORT
-    vMsg pidFile: $pidFile
-    vMsg port: $port
```

Comment:
Probably when I started to develop the test was with an old version.
I have proved with the file that is on the  develop branch, and it works, so I  have restored the same file.

Fixed in 0a207f6